### ISSUE TITLE
[Feature Request]: Inconsistency with trending UI

### ISSUE THREAD

**Issue by efb4f5ff-1298-471a-8973-3d47447115dc** [FreeTubeApp/FreeTube#3205]
### Guidelines

- [X] I have searched the issue tracker for [open](https://github.com/FreeTubeApp/FreeTube/issues?q=is%3Aopen+is%3Aissue) and [closed](https://github.com/FreeTubeApp/FreeTube/issues?q=is%3Aissue+is%3Aclosed) issues that are similar to the feature request I want to file, without success.
- [X] I have searched the [documentation](https://docs.freetubeapp.io/) for information that matches the description of the feature request I want to file, without success.
- [X] This issue contains only one feature request.

### Problem Description

In https://github.com/FreeTubeApp/FreeTube/issues/3204 u see that u hover over an tab it will show an grey underline. On the trending page this doesnt happen. 

### Proposed Solution

When u hover on a tab on the trending page the tab that isnt active should show an underline just like on the channels page.

### Alternatives Considered

None.

### Issue Labels

visual improvement

**Comment by absidue** [comment]
Currently the tabs are implemented very differently on the trending and channel pages, we should probably make them a lot more similar.

**Comment by ChunkyProgrammer** [comment]
Yeah, a shared component for tabs would be cool to have if possible
